src/components/TeamSection/TeamSection.tsx

```diff
@@ -25,15 +24,15 @@ const TeamSection = () => {
           onClick={() => setToggled(prev => !prev)}
         />
         <S.ToggleButtonLabel htmlFor="checkbox">
-          <S.PMText>
+          <S.TeamCategoryText>
             Project Managers &emsp; &emsp; &emsp;&emsp;&emsp; Executives
             <S.Slider></S.Slider>
```

Comment:
The shadow for the selected element is a bit low for this Slider in mobile view (desktop is perfect)
![image](https://github.com/techstartucalgary/tsu-website/assets/10693516/b1a8bc88-f8f1-4b38-bc13-1862d5fd5b70)

Reply:
Good catch! This should be fixed now, please double check